### DISCUSSION TITLE
Chint Dtsu 666 (für Hoymiles) mit Open Dtu nutzen

### DISCUSSION THREAD

**Discussion by rythm1955** [tbnobody/OpenDTU#1305] [Q&A]
Ist es möglich die Daten der DTSU 666 einzulesen und z.B. zur Einspeisesteuerung mit Open Dtu zu verwenden?

**Comment by DPO99** [comment]
Ja das ist Grundsätzlich möglich, da der Zähler wohl über eine RS485 Modbus und eine Impusschnittstelle verfügt. Aber die DTU kann das nicht per default einfach so auslesen und damit die Leistung steuern. Du bräuchtest etwas wie ioBroker oder HomeAssistant um den Zähler auszulesen und dann die DTU zu steuern. Das machen übrigens viele so, also durchaus gängig dein Vorgehen.
